config/initializers/new_framework_defaults_7_1.rb

```diff
@@ -206,7 +206,7 @@
 # leave this optimization off on the first deploy, then enable it on a
 # subsequent deploy.
 #++
-# Rails.application.config.active_record.marshalling_format_version = 7.1
+Rails.application.config.active_record.marshalling_format_version = 7.1
```

Comment:
@myabc 
Assuming there are other migrations in 16.1 that make old workers incompatible with the new database, we wouldn't be doing a rolling upgrade anyway, but have a downtime. In this case, if I understand this correctly, this would not be an issue.